LICENSE

```diff
@@ -1,6 +1,7 @@
 BSD 3-Clause License
 
-Copyright (c) 2020, Philip Meier
+Copyright (c) 2020-2024, Philip Meier
+Copyright (c) 2024-, Slicer Community
```

Comment:
Going by the suggestion from https://github.com/pmeier/light-the-torch/issues/152#issuecomment-2543990886. [SlicerPyTorch uses](https://github.com/fepegar/SlicerPyTorch/blob/e1c85cc0a62e92fa6c38223cd88c611906adb60e/LICENSE#L13-L15)

> This Agreement covers contributions to and downloads from the 3D
Slicer project ("Slicer") maintained by The Brigham and Women's
Hospital, Inc. ("Brigham")

LMK what you prefer.

>

Reply:
@jcfr, thoughts on "Slicer Community" being listed here? I don't think we necessarily need to bring Brigham into the conversation here as this is not directly Slicer related or adopting the Slicer license.

Reply:
In practice, copyright needs to be associated with a legal entity. So here are some options:

```
Copyright (c) 2024-, Brigham and Women's Hospital (BWH)
```

or 

```
Copyright (c) 2024-, Kitware, Inc.
Copyright (c) 2024-, Revvity, Inc.
```

That said, we could also consider specifying copyright as significant contribution are made ?

cc: @lassoan @pieper